drivers/LVHDSR.py

```diff
@@ -1806,6 +1804,11 @@ def delete(self, sr_uuid, vdi_uuid):
         if self.sr.lvActivator.get(self.uuid, False):
             self.sr.lvActivator.deactivate(self.uuid, False)
 
+        try:
+            self.sr.lvmCache.remove(self.lvname)
+        except e:
+            util.SMlog("Failed to remove the volume (maybe is leaf coalescing) for %s err:%d" % (self.uuid, e.errno))
```

Comment:
Also too long, this fits into 80 chars

`            util.SMlog(
                "Failed to remove the volume (maybe is leaf coalescing) "
                "for %s err:%d" % (self.uuid, e.errno))
`